plugin/package.mill

```diff
@@ -35,21 +34,21 @@ trait Plugin extends CrossSbtModule with ScalafmtModule with release.ChiselPubli
   // Does not use common options because -Wconf wasn't supported in early 2.13, nor was "-release:"
   override def scalacOptions = super.scalacOptions() ++ releaseArg(crossScalaVersion)
 
-  def scalaLibraryIvy = v.scalaLibrary(crossScalaVersion)
-  def scalaReflectIvy = v.scalaReflect(crossScalaVersion)
-  def scala2CompilerIvy: Dep = v.scala2Compiler(crossScalaVersion)
-  def scala3CompilerIvy: Dep = v.scala3Compiler(crossScalaVersion)
+  def scalaLibraryMvn = v.scalaLibrary(crossScalaVersion)
+  def scalaReflectMvn = v.scalaReflect(crossScalaVersion)
+  def scala2CompilerMvn: Dep = v.scala2Compiler(crossScalaVersion)
+  def scala3CompilerMvn: Dep = v.scala3Compiler(crossScalaVersion)
```

Comment:
Thanks for updating the method names even though this was not necessary. 👍

Reply:
FWIW it was sort of an accident, I updated these with `sed` 😂